src/systems/ec/implementations/react/components/description-list/src/DescriptionListWithData.jsx

```diff
@@ -32,10 +32,10 @@ const DescriptionListWithData = ({ items, className, ...props }) => {
         }
 
         return (
-          <Fragment key={item.term}>
+          <div key={item.term}>
```

Comment:
adding extra div item here breaks the design.
the `key` property is there only for React, so I'm not sure it is an issue to have it on a Fragment (this seems to be a standard way to handle this https://medium.com/@yurachoi/react-js-fragments-175290da6435)

Reply:
I used the resource provided and put back the `React.Fragment` version which does not trigger linting rules.